src/components/Scaler/index.tsx

```diff
@@ -43,13 +84,23 @@ const findItemsWithChangedDomains = (previousItems, currentItems) => {
   }, []);
 };
 
-export const stripPlaceholderDomain = domain => {
+export const stripPlaceholderDomain = (domain: Domain): Domain | undefined => {
   if (isEqual(PLACEHOLDER_DOMAIN, domain)) {
     return undefined;
   }
   return domain;
 };
 
+const isEqual = (a: Domain, b: Domain): boolean => {
```

Comment:
What's up with `lodash.isequal`?

Reply:
Dunno, TypeScript was unhappy with it and I realized we're only using it to compare two Domains -- we can reimplement that pretty easily. 🤷‍♂️